### DISCUSSION TITLE
【投票】🤔你的体验如何呢？

### DISCUSSION THREAD

**Discussion by ForteScarlet** [simple-robot/Discussions#2] [投票箱]
你是否已经在使用simbot3进行开发了呢？如果是的话，你觉得开发体验如何呢？

**Comment by cuicanontop** [comment]
感觉比2.x繁琐不少

**Comment by Cofking** [comment]
跟着你官网的快速开始，哥们我连Application都启动不起来，@EnableSimbot怎么都找不到，交流群都没。

**Reply by Cofking** [reply]
可能我太菜了

**Reply by ForteScarlet** [reply]
遇到问题及时反馈或去搜搜就可能会发现 https://github.com/orgs/simple-robot/discussions/13 说的也是这个问题，交流群解决不了哥们的问题

**Reply by Cofking** [reply]
跟着这个链接确实解决了。搞了一下午没解决心态爆炸。还不是很会使用git反馈。我的.
